formatter.js

```diff
@@ -0,0 +1,54 @@
+/*
+  Custom eslint formatter for eslint-plugin-i18n-json to allow better error message display.
+  Heavily inspired from https://github.com/sindresorhus/eslint-formatter-pretty.
+*/
+
+const chalk = require('chalk');
+const plur = require('plur');
+const logSymbols = require('log-symbols');
+const indentString = require('indent-string');
+const path = require('path');
+
+const CWD = process.cwd();
+
+const formatter = (results) => {
+  const total = {
+    errors: 0,
+    warnings: 0,
+  };
+  const outputs = results.map(({
```

Comment:
what are `outputs`? should we name it `errorLines`, or something more descriptive?

Reply:
agreed, will make it more descriptive.